src/testing/docker/default-docker-test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+#### build image
+docker build -t fossology/fossology-test . 
+
+#### create network
+docker network create --subnet=172.18.0.0/16 fossology-testnet
+
+#### create container and set IP
+docker create -p 8081:8081 --name fossology-test --net fossology-testnet --ip 172.18.0.22 fossology/fossology-test
+
+#### start container
+docker start fossology-test
+
+#### is container running
+docker ps | grep -q fossology/test
+
+#### fossology needs up to 15 seconds to startup
+sleep 15
+
+#### is fossology reachable? --> check title
+curl -L -s http://172.18.0.22:8081/repo/ | grep -q "<title>Getting Started with FOSSology</title>"
```

Comment:
If you access fossology directly over its IP you need to use port `80` instead of `8081`.